solidity/migrations/2_deploy_contracts.js

```diff
@@ -57,6 +58,11 @@ const all = [
 
 module.exports = (deployer, network, accounts) => {
   deployer.then(async () => {
+    if (network == "keep_dev" || network == "local") {
+      // For keep_dev and local, replace constants with testnet constants.
+      all[all.indexOf(TBTCConstants)] = TBTCDevelopmentConstants
```

Comment:
A little confused by what's happening here. 🤔

 - `deployer.link(library, destinations)` links a library to the destination contracts.
 - we deploy the `TBTCConstants` contract [below](https://github.com/keep-network/tbtc/blob/d4af3b25cc146971f1982a75cbd4388d76201e20/solidity/migrations/2_deploy_contracts.js#L80), and link it into the dependent contracts (eg. TBTCSystem). The TBTCConstants that is linked is the original, TBTCConstants?
 - You've (re)set TBTCConstants in the `all` array, which is the array of contracts with unresolved libraries/symbols.

Reply:
Hah. I missed the deployment line in the update 🤦‍♂️  will fix.